pkg/minikube/cni/cni.go

```diff
@@ -132,42 +132,44 @@ func IsDisabled(cc config.ClusterConfig) bool {
 		return true
 	}
 
-	if chooseDefault(cc).String() == "Disabled" {
+	def, err := chooseDefault(cc)
+	if err == nil && def.String() == "Disabled" {
 		return true
 	}
 	return false
 }
 
-func chooseDefault(cc config.ClusterConfig) Manager {
+func chooseDefault(cc config.ClusterConfig) (Manager, error) {
 	// For backwards compatibility with older profiles using --enable-default-cni
 	if cc.KubernetesConfig.EnableDefaultCNI {
 		klog.Infof("EnableDefaultCNI is true, recommending bridge")
-		return Bridge{}
-	}
-
-	if driver.BareMetal(cc.Driver) {
```

Comment:
is this change related to this PR ?

Reply:
yeah, it removed some weird special case for "none" driver - which needs CNI, just like all other drivers